stable/consul/templates/consul-service.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
```

Comment:
The service used to be wrapped in `{{- if .Values.uiService.enabled }}`. Why was that removed?

Reply:
It wasn't removed. This isn't the Consul UI Service.  That one is under `templates/consul-ui-service.yaml` and is still wrapped in the conditional.